src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.tsx

```diff
@@ -31,8 +40,9 @@ const VideoSpotlightBlock = ({
   button,
   imageURL,
   playButtonOnClick,
-  blackBackground,
-}) => {
+  blackBackground = false,
+  playIcon,
```

Comment:
I don't think this is the right approach, to add the `playIcon` to the props of the component like this. I think it's unlikely that we'd want to change this for each instance of the Video Spotlight Block, so it's better to just bake it into the component definition itself.

If you were hitting issues with TypeScript when importing SVG files, you might need a little snippet of code to add to our TypeScript module declaration file, which I had added in https://github.com/ShelterTechSF/sheltertech.org/pull/207 (which has not yet merged). I made that change in its own commit, so it should be easy to cherry-pick into this PR if you want.